<?xml version="1.0"?>
<launch>
  <param name="path_topic" type="string" value="path" />
  <!--param name="goal_topic" type="string" value="baseline_wp" /-->
  <param name="goal_topic" type="string" value="goal_points" />
  <param name="matrice_name" type="string" value="matrice" />
  <param name="world_name" type="string" value="world" />
  <param name="goal_name" type="string" value="goal" />
  <param name="filter_alpha" type="double" value="0.5" />
  <param name="relative_pose" type="bool" value="true" />
  <param name="point_tf" type="double" value="1.5" />
  <param name="model_path" type="string" value="/home/drone/nvme0/repos/python3_ws/src/orange_picking/useful_models/rgbds_rp_1pt.pth.tar"/>
  <param name="num_phases" type="int" value="4"/>
  <param name="num_points" type="int" value="1"/>
  <param name="num_images" type="int" value="1"/>
  <param name="enable_seg" type="bool" value="True"/>
  <param name="enable_depth" type="bool" value="True"/>
  <param name="states_v" type="bool" value="False"/>
  <param name="states_orange" type="bool" value="False"/>
  <param name="states_rp" type="bool" value="True"/>
  <param name="phase_end" type="bool" value="False"/>
  <node pkg="orange_picking" type="odometry_node" name="odometry" output="screen" />
  <node pkg="orange_picking" type="states_image_inference.py" name="inference" output="screen" />
  <node pkg="orange_picking" type="trajectory_node_cpp" name="trajectory" output="screen"/>
  <!--node pkg="tf" type="static_transform_publisher" name="world_broadcaster" args="0 0 0 0 0 0 1 world optitrak 10"/-->
  <!--node pkg="tf" type="static_transform_publisher" name="camera_broadcaster" args="0 0 0 0 0 0 1 matrice camera_link 10"/-->
  <!--node pkg="rviz" type="rviz" name="rviz" args="-d $(find orange_picking)/rviz/v1.rviz"/-->
  <!--node pkg="tf" type="static_transform_publisher" name="matrice_broadcaster" args="1 1 1 0 0 0 1 world matrice 10"/-->
</launch>
